sdk/storage/azure-storage-file/apireview/azure/storage/file/_deserialize.pyi

```diff
@@ -0,0 +1,14 @@
+# Stubs for azure.storage.file._deserialize (Python 3)
```

Comment:
These methods are exposed due to the fact that they are imported as public names into `azure.storage.file.share_client`.

Reply:
#7798